<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1451</width>
    <height>765</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QQuickWidget" name="quickWidget_MapView">
    <property name="geometry">
     <rect>
      <x>-60</x>
      <y>-20</y>
      <width>1591</width>
      <height>951</height>
     </rect>
    </property>
    <property name="resizeMode">
     <enum>QQuickWidget::ResizeMode::SizeRootObjectToView</enum>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>951</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="numCars">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Demarrer la simulation</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pauseButton">
       <property name="text">
        <string>Pause</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>nettoyer la map</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="toggleGridButton">
       <property name="text">
        <string>Afficher/Cacher Grille</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>0</y>
      <width>371</width>
      <height>751</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWidget" name="widget" native="true">
       <widget class="QWidget" name="panelWidget" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>-50</y>
          <width>371</width>
          <height>791</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string>
                background-color: rgba(0, 0, 0, 180);
                color: white;
            </string>
        </property>
        <widget class="QListWidget" name="logListWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>50</y>
           <width>371</width>
           <height>741</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string>
                background-color: rgba(0, 0, 0, 60);  /* Black with transparency */
                color: white;  /* White text */
            </string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1451</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
